components/linear/actions/create-issue/create-issue.mjs

```diff
@@ -11,5 +11,66 @@ export default {
   key: "linear-create-issue",
   description: "Create an issue (OAuth). See the docs [here](https://developers.linear.app/docs/graphql/working-with-the-graphql-api#creating-and-editing-issues)",
   version: "0.3.7",
```

Comment:
By the way, I forgot to mention, the component's version has to be incremented when changes are made, so that users may update their workflows and receive these changes.

```suggestion
  version: "0.3.8",
```

The app's `package.json` file should also have its version incremented.